dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProcessDefinitionService.java

```diff
@@ -732,26 +746,27 @@ private boolean checkTaskHasSubProcess(String taskType) {
 
     /**
      * import process definition
-     * @param loginUser login user
-     * @param file process metadata json file
+     *
+     * @param loginUser          login user
+     * @param file               process metadata json file
      * @param currentProjectName current project name
      * @return import process
      */
     @Transactional(rollbackFor = Exception.class)
     public Map<String, Object> importProcessDefinition(User loginUser, MultipartFile file, String currentProjectName) {
         Map<String, Object> result = new HashMap<>(5);
         String processMetaJson = FileUtils.file2String(file);
-        List<ProcessMeta> processMetaList = JSON.parseArray(processMetaJson,ProcessMeta.class);
+        List<ProcessMeta> processMetaList = JSON.parseArray(processMetaJson, ProcessMeta.class);
```

Comment:
this JSON come from fastjson

Reply:
> this JSON come from fastjson

the import about fastjson line could be deleted

Reply:
Yes, I didn't fix all this kind of issues at this time. I'll do it next week. :)